src/stcal/alignment/util.py

```diff
@@ -302,19 +308,36 @@ def _calculate_new_wcs(wcs: gwcs.wcs.WCS,
         transform=transform,
         input_frame=wcs.input_frame,
     )
-    axis_min_values, output_bounding_box = _get_axis_min_and_bounding_box(footprints, wcs_new)
-    offsets = _calculate_offsets(
+    axis_min_values, bbox = _get_axis_min_and_bounding_box(footprints, wcs_new)
+    offsets, shifts = _calculate_offsets(
         fiducial=fiducial,
         wcs=wcs_new,
         axis_min_values=axis_min_values,
         crpix=crpix,
     )
 
+    if crpix is None:
+        output_bounding_box = bbox
+    else:
+        output_bounding_box = []
+        for axis_range, minval, shift in zip(bbox, axis_min_values, shifts):
+            output_bounding_box.append(
+                (
+                    axis_range[0] + shift + minval,
```

Comment:
In the docstring for `_calculate_offsets` it says `shifts` are to be subtracted, but they are added here.  Is the doctrstring wrong or is the calculation wrong?

Reply:
Neither is wrong: basically `shifts` are equivalent to 0-indexed CRPIX and should be subtracted from input coordinates to the transformation. However, here we are adjusting the bounding box coordinates (not input coordinates to the WCS transform) so that it "moves" with the CRPIX value. Currently bounding box stays glued to the (0, 0) corner regardless of the output image shape or CRPIX location.